### DISCUSSION TITLE
Remove background from batch TGAs

### DISCUSSION THREAD

**Discussion by NewEXE** [ImageMagick/ImageMagick#1948] [General] [answered]
Hello! I need some help.

So I have many Targa images like this ones: [Google drive link](https://drive.google.com/open?id=1awO0N5sgGdlfYK4A2H7Sr8beHKVYHkNW) (I don't know a good image hosting for TGA, sorry for that).

I just need to remove green background from this images (from all images in dir and subdirs if it's possible). Thank you in advance for your help!

**Comment by fmw42** [comment]
That is a very hard since your images have green foregrounds. You can try using -fuzz xx% -floodfill in one of the background regions.

**Reply by NewEXE** [reply]
Dear Fred, thank you for answer! I'm will try to use you script http://www.fmwconcepts.com/imagemagick/color2alpha/index.php for that task on the weekend. I'm a little confused about task's hardcore level, because Photoshop's Magic Eraser Tool do this task good. Whatever, thank you, I'll left message here if I find a solution.

**Comment by NewEXE** [comment, marked as answer]
Works as I need with
```
convert tree23.tga -fuzz 0% -fill none -draw "matte 0,0 replace" new.tga
```
`-draw` param for each tga can be different (`0,0` part). Then I will write script with `convert`'s options for each tga individually and task is done.
